src/test/java/com/antondoddo/production/usecase/AddUseCaseTest.java

```diff
@@ -0,0 +1,65 @@
+package com.antondoddo.production.usecase;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertSame;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+import com.antondoddo.production.Production;
+import com.antondoddo.production.ProductionObjectMother;
+import com.antondoddo.production.repository.Repository;
+import com.antondoddo.production.repository.exception.CouldNotAddProductionException;
+import java.util.UUID;
+import junitparams.JUnitParamsRunner;
+import junitparams.Parameters;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+@RunWith(JUnitParamsRunner.class)
+public class AddUseCaseTest {
+
+  protected Object[] shouldAddProductionData() {
+    UUID id1 = UUID.randomUUID();
+    UUID id2 = UUID.randomUUID();
+    return new Object[]{
+        new Object[]{
+            ProductionObjectMother.createEpisode(id1)
```

Comment:
Qui potevi anche utilizzare questo metodo:
```suggestion
            ProductionObjectMother.createEpisode()
```
Il quale crea l'ID per te senza passarlo